assets/queries/cloudFormation/security_group_rule_without_description/query.rego

```diff
@@ -0,0 +1,69 @@
+package Cx
+
+CxPolicy [ result ] {
+	  resource := input.document[i].Resources[name]
+    resource.Type == "AWS::EC2::SecurityGroup"
+    
+    object.get(resource.Properties, "GroupDescription", "undefined") == "undefined"
+
+
+    result := {
+                "documentId": 		    input.document[i].id,
+                "searchKey": 	        sprintf("Resources.%s.Properties", [name]),
+                "issueType":          "MissingAttribute",
+                "keyExpectedValue":   sprintf("Resources.%s.Properties.GroupDescription is set", [name]),
+                "keyActualValue": 	  sprintf("Resources.%s.Properties.GroupDescription is undefined", [name])
+              }
+}
+
+CxPolicy [ result ] {
+	resource := input.document[i].Resources[name]
+    resource.Type == "AWS::EC2::SecurityGroup"
+
+    properties := {"SecurityGroupIngress", "SecurityGroupEgress"}
+    object.get(resource.Properties, properties[index], "undefined") == "undefined"
+
+
+    result := {
+                "documentId": 		    input.document[i].id,
+                "searchKey": 	        sprintf("Resources.%s.Properties", [name]),
+                "issueType":		      "MissingAttribute",
+                "keyExpectedValue":   sprintf("Resources.%s.Properties.%s is set", [name, properties[index]]),
+                "keyActualValue": 	  sprintf("Resources.%s.Properties.%s is undefined", [name, properties[index]])
+              }
+}
```

Comment:
I think you can discard this case because you only need to check what is defined without a description. By considering this case, you are forcing a security group to define a "SecurityGroupIngress" and a "SecurityGroupEgress".